docs/diagrams/src/PostFreesurferPipeline-long.GeneralScheme.puml

```diff
@@ -0,0 +1,41 @@
+@startuml PrePostFreesurferPipeline-long general scheme
+|p| Longitudinal TP
+|t| Template
+
+|p|
+-[hidden]->
+group Timepoint processing Stage 1
+:Run FreeSurfer2CaretConvertAndRegisterNonlinear;
+#palegreen: MNI152/Native/MSMSulc
+/{L|R}.sphere_rot.surf.gii;
+note
+generate average 
```

Comment:
What is an the "average surface" *within* the long TP?

Reply:
see comment below